### PR TITLE
fix: remove default dynamics

### PR DESCRIPTION
**Pull request by redeboer**
Implements https://github.com/ComPWA/expertsystem/pull/395 (release 0.6.4) and removes the default dynamics.
https://github.com/ComPWA/tensorwaves/blob/ca494ea48a552b65cf1de40fb88e188ca20450dc/src%2Ftensorwaves%2Fphysics%2Fhelicity_formalism%2Famplitude.py#L159-L160

That was an implementation that seemed convenient, but modified the dynamics section of the input `AmplitudeModel`. 

Note that a long-term fix should come through #172 and https://github.com/ComPWA/expertsystem/issues/22

### CODE REVIEW

**Comment by redeboer** [comment]
Note: 10e115e has to be reverted once https://github.com/ComPWA/expertsystem/pull/395 is through and `expertsystem==0.6.4` is released

File: tests/recipe/test_amplitude_creation.py [hunk 1]
```diff
@@ -33,7 +33,7 @@ def test_helicity(helicity_model: es.AmplitudeModel):
 
     builder = IntensityBuilder(model.particles, kinematics, phsp_sample)
     intensity = builder.create_intensity(model)
-    assert len(intensity.parameters) == 9
+    assert len(intensity.parameters) == 10
```
Comment (spflueger) [inline tests/recipe/test_amplitude_creation.py:36]: why the parameters go up by 1?

Comment (redeboer) [reply tests/recipe/test_amplitude_creation.py:36]: Because the meson radius is written as a parameter as well now

Comment (redeboer) [reply tests/recipe/test_amplitude_creation.py]: Oh no, forget that: d2d1b90
This also came by in #159, I think

Comment (redeboer) [reply tests/recipe/test_amplitude_creation.py]: This makes it a bit more transparent c666dd8

File: tests/recipe/test_amplitude_creation.py [hunk 2]
```diff
@@ -33,7 +33,17 @@ def test_helicity(helicity_model: es.AmplitudeModel):
 
     builder = IntensityBuilder(model.particles, kinematics, phsp_sample)
     intensity = builder.create_intensity(model)
-    assert len(intensity.parameters) == 9
+    assert set(intensity.parameters) == {
+        "strength_incoherent",
+        "Position_J/psi(1S)",
+        "Width_J/psi(1S)",
```
Comment (spflueger) [inline tests/recipe/test_amplitude_creation.py:39]: Is it intended that the J/psi has a BreitWigner dynamics?

Comment (redeboer) [reply tests/recipe/test_amplitude_creation.py:39]: I wouldn't say intended, but currently, mass and width are always registered, no matter what the dynamics are...
https://github.com/ComPWA/tensorwaves/blob/d2d1b90a26b532bc16a132f538a43153e2e87426/src%2Ftensorwaves%2Fphysics%2Fhelicity_formalism%2Famplitude.py#L591-L632

I would say this should be properly tackled in e.g. #162. This PR just removes the default dynamics (particularly the buggy way in which it was implemented).

**Review by spflueger — APPROVED** [review]

**Pull request merged**